src/Raven.Client/Documents/Operations/AI/Agents/ToolQuery.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Linq;
+using Raven.Client.Documents.Session;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.AI.Agents
+{
+    public class ToolQuery : IDynamicJson
+    {
+        public static ToolQuery Build<T>(string name, string description, IRavenQueryable<T> query)
```

Comment:
this will be removed for now, so let's add a ctor

Reply:
Done